test/evergreen/setup_spawn_host.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+#
+# The script is used to setup the environment for a spawn host. Currently the script 
+# unzips the test artifacts, prepend the toolchain path to the PATH environment and exports the
+# LD_LIBRARY_PATH.
+# 
+# Once setup, the script will notify the user who spawned the host that it is finished.
+#
+cd $HOME || exit 1
+TOOLCHAIN_ROOT=/opt/mongodbtoolchain/v4
+
+# Communicate to users that logged in before the script started that nothing is ready.
+wall "The setup_spawn_host script has just started setting up the debugging environment."
+
+# Make a directory on the larger volume. Soft-link it under the home directory.
+mkdir -p /data/wiredtiger
+ln -s /data/wiredtiger .
+cd wiredtiger || exit 1
+
+# Find the test artifacts.
+WT_ARCHIVE=$(ls /data/mci/artifacts-*/*.tgz | grep -v "compile" 2>/dev/null)
+if [[ -n $WT_ARCHIVE ]]; then
+    tar --wildcards -xzf $WT_ARCHIVE
```

Comment:
One possible issue here could be that WT_ARCHIVE may have an array of values. We wish to only take the first one. Otherwise it may create issues later.

Reply:
True, i have added a HEAD command to fix this issue.